app/views/kpis/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id="v-app">
+  <kpi-page
+    endpoint="/kpis/kpi_endpoint"
```

Comment:
Yes, it would be `kpis_kpi_endpoint_path` but that's just as verbose without really helping - also I'm injecting ERB into the prop which I wanted to avoid.

Reply:
I've started to become averse to injecting Ruby as props in Vue components as that's what we currently do, but the industry practice seems to be to treat Rails as a backend that we query in methods within the component instead of directly piping Ruby variables into the props and having them available there. I wanted to start practicing decoupling of the FE from the BE with this feature. Also, I've started to try and avoid too much ERB in the view files as it clutters them up. 

You're right about the single source of truth though - if the endpoint was changed for whatever reason then this component will break. I've forgotten that you can alias routes, so provided that alias isn't changed then changing the relative path should be OK. Will fix